design-toolkit/components/src/components/color-picker/index.tsx

```diff
@@ -136,29 +136,26 @@ function ColorSwatchPickerItem({
 
 const ColorMapContext = createContext<Map<string, Color> | null>(null);
 
-export interface ColorPickerProps extends AriaColorSwatchPickerProps {
-  options: AriaColorSwatchPickerItemProps['color'][];
-  ref?: ForwardedRef<HTMLDivElement>;
-}
-
-interface ColorSwatchPickerProps extends Omit<ColorPickerProps, 'options'> {
-  children?: ReactNode;
-}
-
 function ColorSwatchPicker({
   ref = null,
   className,
+  classNames,
   children,
   ...props
 }: ColorSwatchPickerProps) {
   [props, ref] = useContextProps(props, ref, ColorSwatchPickerContext);
   const state = useColorPickerState(props);
   const colorMap = useMemo(() => new Map(), []);
+  const { layout = 'grid', ...rest } = props;
+
   return (
     <AriaListBox
-      className={cn('flex flex-wrap gap-s', className)}
-      style={props.style}
-      layout={props.layout || 'grid'}
+      {...rest}
+      className={composeRenderProps(
+        classNames?.picker || className,
```

Comment:
When implementing `classNames` we should be omitting the `className` prop instead of supporting both